biodatasets/scifact/scifact.py

```diff
@@ -0,0 +1,439 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and
+#
+# * Nicholas Broad (nbroad1881)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import json
+from typing import List, Tuple, Dict
+from itertools import chain
+
+import datasets
+from dataclasses import dataclass
```

Comment:
lets drop this import and instead import the `BigBioConfig` 
`from utils.configs import BigBioConfig`